main.go

```diff
@@ -41,6 +49,11 @@ var (
 			Help: "Errors in requests to the SNMP exporter",
 		},
 	)
+	sc = &SafeConfig{
+		C: &config.Config{},
+	}
+	err      error
```

Comment:
Why not? Configuration is global for the application and it should be available in different functions in main.go.